universe/Conditions.h

```diff
@@ -1169,17 +1025,10 @@ struct FO_COMMON_API EmpireStockpileValue final : public Condition {
     EmpireStockpileValue(ResourceType stockpile,
                          std::unique_ptr<ValueRef::ValueRef<double>>&& low,
                          std::unique_ptr<ValueRef::ValueRef<double>>&& high);
-    EmpireStockpileValue(std::unique_ptr<ValueRef::ValueRef<int>>&& empire_id,
```

Comment:
should be implemented, not removed...

Reply:
Why? It is not called anywhere.

Reply:
It should be implemented and called from the parser

Reply:
Maybe, I don't know. The problem is that this is not part of the scope for this PR. I can undo the deletion commit.

Reply:
Removed corrsponding commit.